ios/RockCheckin/Classes/AppDelegate.m

```diff
@@ -118,6 +118,15 @@ - (void)registerUserDefaults
     }
     
     [NSUserDefaults.standardUserDefaults registerDefaults:defaultValues];
+
+    // Set any settings pushed from MDM
+    NSDictionary *serverConfig = [NSUserDefaults.standardUserDefaults dictionaryForKey:@"com.apple.configuration.managed"];
+    if(serverConfig == nil) {
+        serverConfig = @{};
+    }
+    for(id key in serverConfig) {
+        [NSUserDefaults.standardUserDefaults setObject:[serverConfig objectForKey:key] forKey:key];
+    }
```

Comment:
I wasn't sure whether it made more sense to revert to the previous settings, or to keep the last configured settings when a device was un-enrolled. It doesn't matter to me either way since the only time I would be un-enroling a device is to wipe and re-purpose it. If you think it is better to revert, I'm good with that.

I just pushed a new commit implementing a settings helper class that reads the settings from MDM or local, rather than overwriting the local settings. I have tested this by making changes in the settings app, and by pushing changes from Mossyle MDM. The app picks up settings from both locations correctly.